### ISSUE TITLE
Cannot access person info in validations 

### ISSUE THREAD

**Issue by abbyad** [medic/medic-webapp#3013]
We can access a patient's name in a form's auto-reply, but not in a validation.

# Auto Reply
![image](https://cloud.githubusercontent.com/assets/789512/21357292/a2b05222-c6a2-11e6-871a-96edc5eae108.png)

# Validation
![image](https://cloud.githubusercontent.com/assets/789512/21357330/bc6a2788-c6a2-11e6-9abf-b790a060cb31.png)

**Comment by abbyad** [comment]
Relevant excerpt of config:

```
  "registrations": [
    {
      "form": "P",
      "validations": {
        "join_responses": true,
        "list": [
          {
            "property": "last_menstrual_period",
            "rule": "(integer && min(0) && max(42))",
            "message": [
              {
                "content": "The registration format for {{patient_name}} ({{patient_id}}) is incorrect, please ensure that LMP is a number between 0 and 42.",
                "locale": "en"
              }
            ]
          }
        ]
      },
      "messages": [
        {
          "message": [
            {
              "content": "Thank you for registering {{patient_name}} ({{patient_id}}). Their EDD is {{#date}}{{expected_date}}{{/date}}",
              "locale": "en"
            },
```

For the validation I have tried `{{patient_name}}`, `{{name}}`, `{{contact.name}}`, and `{{patient.name}}` -- all give no output.

**Comment by sglangevin** [comment]
@abbyad is this needed for Standard config? Or we have a workaround for that?

**Comment by abbyad** [comment]
Given the recent changes this may actually be resolved... should retest.

**Comment by garethbowen** [comment]
I just tried this on the latest sentinel. The validation message I used is: `The patient name is too long: {{patient_name}}.`. I used the old school messages style not the translation key style but they should both work.

![success](https://cloud.githubusercontent.com/assets/2064938/26564980/3cdfe2ec-453b-11e7-8b29-2e3236be6e64.png)

@abbyad I'll close this and set it to AT so you (or whomever) can confirm.
